f2f-ipv-stub/template.yaml

```diff
@@ -49,7 +49,7 @@ Mappings:
       IPVStubID: "BD7B2A5D"
       JWKSURI: "https://api.review-o.build.account.gov.uk"
       IPVSTUBURL: "ipvstub.review-o.build.account.gov.uk"
-      OAUTHFRONTBASEURI:  "https://review-o.build.account.gov.uk"
+      OAUTHFRONTBASEURI:  "https://www.review-o.build.account.gov.uk"
```

Comment:
I think adding this would break browser tests [here](https://github.com/alphagov/di-ipv-cri-f2f-front/blob/121ab032b4a0b5af786cc92a137b20e46366557b/test/browser/pages/relying-party.js#L14)

Reply:
🙏 Updated browser tests [here](https://github.com/alphagov/di-ipv-cri-f2f-front/pull/94)